test/kind/config.yaml

```diff
@@ -5,3 +5,16 @@ nodes:
 - role: worker
 - role: worker
 - role: worker
+# These apiServer settings are included for running the CSI provider on K8s
+# prior to 1.21
+kubeadmConfigPatches:
+  - |
+    apiVersion: kubeadm.k8s.io/v1beta2
+    kind: ClusterConfiguration
+    metadata:
+      name: config
+    apiServer:
+      extraArgs:
+        "service-account-issuer": "kubernetes/serviceaccount"
```

Comment:
```suggestion
        "service-account-issuer": "https://kubernetes.default.svc.cluster.local"
```

I would err towards providing a non-legacy value unless I'm missing something.

Reply:
Yep, agreed. Done in 01af2f52355bd1f3960e19e1ed6a32e7975fcd72.